content/other_applications/member_check.md

```diff
@@ -12,15 +12,15 @@ The member check can be found at [utn.se/medlemskoll](https://www.utn.se/medlems
 ## API
 
 The member check is also available as an api and can be used from a website, app etc.
-To use the api, send a **POST** request to `https://old.utn.se/member_check/check_membership.php`.
+To use the api in Moore, send a **POST** request to `/member_check_api/`.
 
 ### Params
-
-In the **POST** request, you must send the social security number (personnummer) of the person you want to perform a member check on.
+In the **POST** request you must send a JSON object with key/value pair: ssn/'personnummer'. 
```

Comment:
Just to make it a bit more clear
```suggestion
In the **POST** request you must send a JSON object with key/value pair: ssn/'personnummer_of_the_person'. 
```

Also add that the content type must be set to "application/json"

Reply:
ok! fixed in https://github.com/UTNkar/documentation/pull/3/commits/bb4249c70fa8f8a27e56d1686757449667c948f3